examples/src/migration/README.rst

```diff
@@ -0,0 +1,19 @@
+##################
+Migration Examples
+##################
+
+The `Encryption SDK for Python`_ now uses the `AWS Cryptographic Material Providers Library`_,
+which introduces keyrings in place of Master Key Providers. The MPL abstracts lower
+level cryptographic materials management of encryption and decryption materials.
+
+This directory contains some examples to migrate from the old version of the ESDK.
```

Comment:
Ah, no -- the first 3 are for MKP -> KMS. The last one is for the version upgrade

I'd suggest keeping the first 3 in a group explaining "MKP to KMS" and separate the last one as "1.x to newer version upgrade using keyrings"

Reply:
Sounds good. Thanks!